### ISSUE TITLE
It's almost impossible to do task 4

### ISSUE THREAD

**Issue by Yuhan-Ut** [alecjacobson/computer-graphics-text-to-image#13]
Task 4 want 6 image about a (un-popular) character in same style
and after the 30 min+ generating, when we want another image with the same prompt, the model is already changed, and we get extremely  different character
![image-3](https://github.com/alecjacobson/computer-graphics-text-to-image/assets/113068495/db9c5849-d0bb-4752-96c4-34fcb1d6d5ab)
![image](https://github.com/alecjacobson/computer-graphics-text-to-image/assets/113068495/5183b87a-045d-459c-9dd5-48ee5cfa1c45)
this is 2 image I got with same seed, almost same prompt

**Comment by Yuhan-Ut** [comment]
Can I use other deployed online sd tools?
for example, I found this one
https://stablediffusionweb.com/#ai-image-generator

**Comment by wenzhi-guo** [comment]
Yes, after discussing with the teaching team,  it is fine to use another online tool (but the online tool needs to use the same model versions as we have in our web app). The specific site above is OK to use for this assignment. Please indicate in your submission somewhere what specific site you used for this assignment.

**Comment by hzc011213** [comment]
Can I use local stable diffusion web UI with the same model versions as in the web app?

**Comment by hzc011213** [comment]
Can I use local stable diffusion web UI with the same model versions as in the web app? If so what sampling method (and other parameters as shown below) should I use?
<img width="726" alt="image" src="https://github.com/alecjacobson/computer-graphics-text-to-image/assets/64294214/e12c89f0-71de-4055-bc0f-2865f3cb4250">

**Comment by JustinZ17** [comment]
Could I replace the "model" section of the JSON from "[SDv1.5|SDv2.1|SDXL]" with the link to the site we are using for this assignment?

**Comment by wenzhi-guo** [comment]
> Can I use local stable diffusion web UI with the same model versions as in the web app? If so what sampling method (and other parameters as shown below) should I use? <img alt="image" width="726" src="https://private-user-images.githubusercontent.com/64294214/286502582-e12c89f0-71de-4055-bc0f-2865f3cb4250.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEyODYwODgsIm5iZiI6MTcwMTI4NTc4OCwicGF0aCI6Ii82NDI5NDIxNC8yODY1MDI1ODItZTEyYzg5ZjAtNzFkZS00MDU1LWJjMGYtMjg2NWYzY2I0MjUwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI5VDE5MjMwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdiZGJlMTdlYWY4MjFmYzBkODkxNzg2MTVhYzA0Njg2MGYwMDUyZjM3MzNhMDUwOTEwNDIxODc2ODExZDdiNDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.IEztnMRbtaYE16Kem2KEXkogxJwojVTaaTP7Vd3iA-s">

Yes, this is OK if the local version is the same as the web app. Use the default setting for the model (for parameters that do not exist in our web app) should be fine.

**Comment by wenzhi-guo** [comment]
> Could I replace the "model" section of the JSON from "[SDv1.5|SDv2.1|SDXL]" with the link to the site we are using for this assignment?

This is for the JSON file you submit for each task? If so, yes, this is ok, but make sure the zip file still passes the validation link provided in the readme file.
